Docs/nit install (#903)
apify/apify-cli: docs/installation.md

irm https://apify.com/install-cli.ps1 | iex
```

:::tip No need for Node.js

If you install Apify CLI using our installation scripts, you don't need to install Node.js to use the CLI. This is because we use [Bun](https://bun.sh/) as a bundler, which creates a standalone executable file. This is especially useful for Python users or anyone who prefers not to manage Node.js dependencies.

:::

## Homebrew

```bash
brew install apify-cli
```

:::tip Node.js required with Homebrew

> If you install Apify CLI using Homebrew, be aware that internally it still installs Node.js, as Homebrew does not support having everything in one executable file.

Reviewer: ```suggestion
If you install Apify CLI using Homebrew, be aware that internally it will also install Node.js. This should not cause issues, but if it does, you might need to adjust your `PATH` environment variable to deprioritize Homebrew's install of Node.js
```

irm https://apify.com/install-cli.ps1 | iex
```

:::tip No need for Node.js

If you install Apify CLI using our installation scripts, you don't need to install Node.js to use the CLI. This is because we use [Bun](https://bun.sh/) as a bundler, which creates a standalone executable file. This is especially useful for Python users or anyone who prefers not to manage Node.js dependencies.

:::

## Homebrew

```bash
brew install apify-cli
```

> :::tip Node.js required with Homebrew

Reviewer: ```suggestion
```

:::tip Homebrew and Node.js dependency

When you install Apify CLI using Homebrew, it automatically installs Node.js as a dependency. If you already have Node.js installed through another method (e.g., `nvm`), this may create version conflicts.

If you experience Node.js version conflicts, modify your `PATH` environment variable to prioritize your preferred Node.js installation over Homebrew's version.

:::

> :::

Reviewer: ```suggestion
```
